packages/eslint-plugin-lit-a11y/lib/rules/interactive-supports-focus.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @fileoverview
+ * @author open-wc
+ */
+
+const includes = require('array-includes');
+const { TemplateAnalyzer } = require('../../template-analyzer/template-analyzer.js');
+const { eventHandlersByType } = require('../utils/eventsHandlers.js');
+const { getAttrVal } = require('../utils/getAttrVal.js');
+const { getTabIndex } = require('../utils/getTabIndex.js');
+const { isDisabledElement } = require('../utils/isDisabledElement.js');
+const { isHiddenFromScreenReader } = require('../utils/isHiddenFromScreenReader.js');
+const { isInteractiveElement } = require('../utils/isInteractiveElement.js');
+const { isPresentationRole } = require('../utils/isPresentationRole.js');
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description:
+        'Elements with an interactive role and interaction handlers (mouse or key press) must be focusable.',
+      category: 'Fill me in',
+      recommended: false,
+    },
+    fixable: null, // or "code" or "whitespace"
+    schema: [
+      // fill in your schema
+    ],
+  },
+
+  create(context) {
+    // variables should be defined here
+
+    //----------------------------------------------------------------------
+    // Helpers
+    //----------------------------------------------------------------------
+
+    const tabbable = (context.options && context.options[0] && context.options[0].tabbable) || [];
+
+    const interactiveProps = [...eventHandlersByType.mouse, ...eventHandlersByType.keyboard];
+
+    //----------------------------------------------------------------------
+    // Public
+    //----------------------------------------------------------------------
+
+    return {
+      TaggedTemplateExpression: node => {
+        if (
```

Comment:
We have a helper in utils, so you can
```js
const { isHtmlTaggedTemplate } = require('../utils/isLitHtmlTemplate.js');
```
and here:
```suggestion
        if (isHtmlTaggedTemplate(node)) {
```